tests/integration/test_study_generator.py

```diff
@@ -0,0 +1,89 @@
+import os
+import pytest
+import requests
+
+from creator.ingest_runs.data_generator.study_generator import (
+    ENDPOINTS,
+    delete_entities,
+)
+from tests.integration.fixtures import (
+    LOCAL_DATASERVICE_URL,
+    test_study_generator,
+    dataservice_setup,
+)
+
+
+def test_delete_entities():
```

Comment:
Remove this once the `delete_entities` function is removed from StudyGenerator since it is being implemented in https://github.com/kids-first/kf-utils-python/pull/24

Reply:
@gsantia This test module can be removed since study generator isn't part of the pr anymore